content/lessons/chapter-2/mining-1.tsx

```diff
@@ -295,6 +296,17 @@ export default function Mining1({ lang }) {
               />
             </Card>
           </div>
+          <Card className="flex" highlight={nonceHighlight}>
+            <TitleCard title={'Hash Rate'} disabled={false}></TitleCard>
+            <BoxDisplayer className="ml-10">
+              <span>25.00%</span>
+            </BoxDisplayer>
+            <HashFrequency
+              disabled={nonce === 0}
+              step={step}
+              hashPower={hashPower}
+            />
+          </Card>
```

Comment:
``` suggestion
```

Reply:
Addressed in 8852c948c97a12026a90871b9b7c7208e8ebd52b
Thanks!